### DISCUSSION TITLE
2010/12/10/%E5%BC%BA%E5%A3%AE%E7%9A%84%E8%B1%86%E8%B1%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#306] [Announcements]
# 2010/12/10/%E5%BC%BA%E5%A3%AE%E7%9A%84%E8%B1%86%E8%B1%86

豆豆妈在怀孕期间吃的相当不错，可是自己居然一点都没长胖，所有营养全给了豆豆和他的小房子。离预产期还有一个半月的时候，豆豆妈去产检估重。医生就说胎儿已经超过六斤了。吓得豆豆妈赶紧控制饭量，并且打定了主意要 剖腹产。产前一星期，医生隔着妈妈肚皮摸了摸豆豆，说超过7斤。 豆豆生下来的时候果然不小：体重七斤三两（3.65kg），身高54cm，双顶径97mm，所有指标都是偏上的。产后的第四天，我抱着豆豆去游泳，别的家长还以为我们豆豆是生了个把月的呢。游泳房的桌子上，光不刺溜躺上一排小宝，豆豆明显比别人大一圈。

https://blog.qizhen.xyz/2010/12/10/%E5%BC%BA%E5%A3%AE%E7%9A%84%E8%B1%86%E8%B1%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

foolooo
2010年12月17日 09:16 编辑
果然当了父亲界面也深沉多了:)

回复
